library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -36,7 +37,17 @@
 
 	public final String currency;
 
-	public final Double priceValue;
+	public final double priceValue;
```

Comment:
Sure, that makes sense logically, but I think we should still avoid breaking changes with API unless there's going to be a new major release.